bower.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "iron-fit-behavior",
-  "version": "0.8.0",
+  "version": "0.9.0",
   "private": true,
-  "main": "iron-fit-behavior",
-  "authors": "The Polymer Authors",
+  "main": [
+    "iron-fit-behavior.html"
+  ],
+  "authors": [
+    "The Polymer Authors"
+  ],
   "repository": {
     "type": "git",
     "url": "git://github.com/PolymerElements/iron-fit-behavior.git"
   },
   "dependencies": {
-    "polymer": "Polymer/polymer#v0.8.0-rc.7"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1"
   },
   "devDependencies": {
-    "test-fixture": "PolymerElements/test-fixture#^0.8.0",
+    "iron-component-page": "PolymerElements/iron-component-page#^0.9.0",
+    "test-fixture": "PolymerElements/test-fixture#^0.9.0",
     "web-component-tester": "*",
     "webcomponentsjs": "^0.6.0"
```

Comment:
not sure if this needs to be `"webcomponents/webcomponentsjs#^0.6.0"` but i guess if it works now...